app/views/shell/_top.html.erb

```diff
@@ -3,7 +3,7 @@
   <head>
     <meta charset="utf-8">
     <% title = yield(:title) %>
-    <title><%= title || "#{ApplicationConfig['COMMUNITY_NAME']} Community" %></title>
+    <title><%= title || "#{community_name} Community" %></title>
```

Comment:
this could probably be `community_qualified_name`

Reply:
oh mannnnnnn 😅 Thanks Rhymes, I fixed the two instances of this that I found. This copy/pasting in the PR is driving me insane 😁